Gems/PhysX/PhysX4/Source/Module.cpp

```diff
@@ -92,27 +92,6 @@ namespace PhysX
                 m_modules.push_back(AZStd::move(sceneCoreModule));
             }
 #endif // defined(PHYSX_EDITOR)
-
-            // Load PhysX SDK dynamic libraries when running on a non-monolithic build.
-            // The PhysX Gem module was linked with the PhysX SDK dynamic libraries, but
-            // some platforms may not detect the dependency when the gem is loaded, so we
-            // may have to load them ourselves.
-#if AZ_TRAIT_PHYSX_FORCE_LOAD_MODULES && !defined(AZ_MONOLITHIC_BUILD)
-            {
-                const AZStd::vector<AZ::OSString> physXModuleNames = { "PhysX", "PhysXCooking", "PhysXFoundation", "PhysXCommon" };
-                for (const auto& physXModuleName : physXModuleNames)
-                {
-                    AZ::OSString modulePathName = physXModuleName;
-                    AZ::ComponentApplicationBus::Broadcast(&AZ::ComponentApplicationBus::Events::ResolveModulePath, modulePathName);
-
-                    AZStd::unique_ptr<AZ::DynamicModuleHandle> physXModule = AZ::DynamicModuleHandle::Create(modulePathName.c_str());
-                    bool ok = physXModule->Load(false/*isInitializeFunctionRequired*/);
-                    AZ_Error("PhysX::Module", ok, "Error loading %s module", physXModuleName.c_str());
-
-                    m_modules.push_back(AZStd::move(physXModule));
```

Comment:
Has it been checked that the 3rdParty PhysX libraries aren't using any static variables which could now be duplicated across DLLs now that the PhysX static libraries can now be linked linked multiple modules in non-monolithic builds.

Reply:
[Initialization of PhysX ](https://gameworksdocs.nvidia.com/PhysX/4.1/documentation/physxguide/Manual/Startup.html) returns a handle, so there shouldn't be any internal state maintained specifically.

The initialization of the library is done in the [Gem](https://github.com/o3de/o3de/blob/bb977c1d71286021566412e606840278481accd1/Gems/PhysX/Code/Source/System/PhysXSystem.cpp#L443 ) code at